### PR TITLE
blah

### PR DESCRIPTION
**Pull request by ghtstr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# trinug
\ No newline at end of file
+# trinug
+
+yay trinug
+
+test 2
```
Comment (twiest) [inline README.md]: Um, I think this is meant to be test 3.

/cc @ghtstr

**Review by twiest — CHANGES_REQUESTED** [review]
Thanks for the contribution. Comment inline.

File: README.md (hunk 1 above)
Comment (ghtstr) [reply README.md]: No, I think it should be 2.

**Review by twiest — APPROVED** [review]
/lgtm

**Pull request merged**